GVRf/Framework/jni/objects/vertex_bone_data.cpp

```diff
@@ -50,8 +50,15 @@ int VertexBoneData::getFreeBoneSlot(int vertexId) {
 }
 
 void VertexBoneData::setVertexBoneWeight(int vertexId, int boneSlot, int boneId, float boneWeight) {
-    boneData[vertexId].ids[boneSlot] = boneId;
-    boneData[vertexId].weights[boneSlot] = boneWeight;
+    if (BONES_PER_VERTEX <= boneSlot || 0 > boneSlot) {
```

Comment:
There is some coding style blog about readability: https://theholyjava.wordpress.com/2012/12/15/what-is-clean-code-quotes. FYI. It'd better not make it less natural for anyone who might read this code.